### PR TITLE
Calculate HPR counter clock wise

### PR DESCRIPTION
**Pull request by ggetz**
Can't push to that branch directly, but additional fixes to  https://github.com/AnalyticalGraphicsInc/cesium/pull/5809

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.



__I am a bot who helps you make Cesium awesome!__ Thanks again.

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Change Log
    * `Scene/OrthographicOffCenterFrustum` has been removed. Use `Core/OrthographicOffCenterFrustum`.
    * `Scene/PerspectiveFrustum` has been removed. Use `Core/PerspectiveFrustum`.
    * `Scene/PerspectiveOffCenterFrustum` has been removed. Use `Core/PerspectiveOffCenterFrustum`.
+* Deprecated
+   * The default behavior of the functions `HeadingPitchRoll.fromQuaternion`, `Matrix3.fromHeadingPitchRoll`, `Quaternion.fromHeadingPitchRoll`, `Transforms.headingPitchRollToFixedFrame`, and `Transforms.headingPitchRollQuaternion` has been deprecated. An optional boolean flag can be supplied to these functions that, if true, uses the classical orientation of heading and pitch calculated counter-clockwise. The flag will be removed and the new behavior made default in 1.40.[#5666](https://github.com/AnalyticalGraphicsInc/cesium/issues/5666)
```
Comment (pjcozzi) [inline CHANGES.md]: Please update this and move it to the breaking change section.

File: Source/Core/HeadingPitchRoll.js [hunk 2]
```diff
@@ -40,17 +42,23 @@ define([
             throw new DeveloperError('quaternion is required');
         }
         //>>includeEnd('debug');
+
+        deprecationWarning('HeadingPitchRoll.fromQuaternion', 'This function now uses a counter-clockwise orientation as per mathematical conventions. This deprecation warning will be removed in Cesium 1.40.');
```
Comment (pjcozzi) [inline Source/Core/HeadingPitchRoll.js]: Submit an issue to remove this warning.

Also, make the warning more precise.  This started in 1.38 and users just need to add the negative sign.

Comment (pjcozzi) [reply Source/Core/HeadingPitchRoll.js]: Same comment throughout.

**Comment by ggetz** [comment]
@pjcozzi Updated. Also opened https://github.com/AnalyticalGraphicsInc/cesium/issues/5862

**Comment by pjcozzi** [comment]
Thanks again @kaktus40 @ggetz!

**Pull request merged**